Distraction Shield/statisticsPage/statistics.html

```diff
@@ -37,6 +37,7 @@
                 <th></th>
                 <th>Site</th>
                 <th>#intercepts</th>
+                <th>Time spent</th>
```

Comment:
now there is a "time spent" for the time spent on exercises and a "time spent" for time spent procrastinating on a specific blacklisted website. Maybe change this "Time spent" to "Time Wasted"? or something in that direction to make a clear distinction between the two.